src/classes/UTIL_OrgTelemetry_BATCH.cls

```diff
@@ -0,0 +1,82 @@
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2018
+* @group Utilities
+* @description A more controllable method to handle the asynchronous submission of Telemetry data to the LMO
+* without having to worry about governor limits when it comes to querying large datasets in LDV orgs. By iterating
+* through a custom List<> of Enum Types with a batch size of ONE, the execute() method can call a service method
+* to handle querying and submitting the specific telemetry data only for that Enum type. If for some reason that
+* one query were to fail, which is theoretically possible in a huge organization where there may be other jobs
+* executing or other factors at play, the rest of this job will continue. That ensures that our LMO collects as much
+* data as possible. In addition, it can help us identify which specific telemetry data processes are failing.
+*/
+public without sharing class UTIL_OrgTelemetry_BATCH implements Database.Batchable<UTIL_OrgTelemetry_SVC.TelemetryTypeEnum> {
+
+    /**
+     * @description List of High Level Telemetry Data Types to be processed asynchronously by this batch job.
+     */
+    private static final List<UTIL_OrgTelemetry_SVC.TelemetryTypeEnum> telemetryTypesList =
+        new List<UTIL_OrgTelemetry_SVC.TelemetryTypeEnum>{
+            UTIL_OrgTelemetry_SVC.TelemetryTypeEnum.NPSP_Settings,
+            UTIL_OrgTelemetry_SVC.TelemetryTypeEnum.Org_Environment,
+            UTIL_OrgTelemetry_SVC.TelemetryTypeEnum.Data_MaxNumRelatedOpps,
+            UTIL_OrgTelemetry_SVC.TelemetryTypeEnum.Data_Count_of_RD_Opps_All,
+            UTIL_OrgTelemetry_SVC.TelemetryTypeEnum.Data_Count_of_RD_Opps_OpenEnded,
+            UTIL_OrgTelemetry_SVC.TelemetryTypeEnum.Data_CountOfErrorLog
+        };
+
+    /**
+     * @description Return the list of Telemetry Type Enum values to be processed by this batch job
+     * @param BC BatchableContext (not used)
+     * @return telemetryTypesList list from above
+     */
+    public Iterable<UTIL_OrgTelemetry_SVC.TelemetryTypeEnum> start(Database.BatchableContext BC) {
+        if (!Test.isRunningTest()) {
+            return UTIL_OrgTelemetry_BATCH.telemetryTypesList;
+        } else {
+            return new List<UTIL_OrgTelemetry_SVC.TelemetryTypeEnum>{UTIL_OrgTelemetry_SVC.TelemetryTypeEnum.Data_CountOfErrorLog};
+        }
+    }
+
+    /**
+     * @description Process the specific set of FeatureParameter submissions to the LMO based on the Telemetry
+     * Type Enum value.
+     * @param info BatchableContext (not used)
+     * @param types List<UTIL_OrgTelemetry_SVC.TelemetryTypeEnum> -- Batch Size of ONE is required
+     */
+    public void execute(Database.BatchableContext info, List<UTIL_OrgTelemetry_SVC.TelemetryTypeEnum> types) {
+        UTIL_OrgTelemetry_SVC telemetrySvc = new UTIL_OrgTelemetry_SVC();
+        telemetrySvc.processTelemetryType(types[0]);
```

Comment:
It's technically possible to check for a batch size > 1 in the execute method, however even as a public method it can only be called by managed code. I'm not sure it's worth adding validation logic to this given that that we're only protecting against our own usage and that should be covered in the class comments.